examples/assemble_blast.json

```diff
@@ -28,13 +28,15 @@
     "rapsearch2_accessions_out": ["nr.refseq.fasta"],
     "refined_gsnap_out": [
       "gsnap.blast.m8",
+      "gsnap.blast.top.m8",
```

Comment:
move this to the end of the json?

Reply:
Agreed, it might be less prone to human error if new outputs are added to the end of the list rather than in the middle. Or just confirm that all the steps that refer to `refined_gsnap_out` still refer to the correct indices of these lists. For example, the postprocess dag in idseq-web (https://github.com/chanzuckerberg/idseq-web/blob/7da56caff3a686bcf2559c51ece28a0f258779c5/app/lib/dags/postprocess.json.erb#L146) indicates that the step combine_taxon_counts (https://github.com/chanzuckerberg/idseq-dag/blob/master/idseq_dag/steps/combine_taxon_counts.py#L12) is pulling the item at index 3 of the `refined_gsnap_out` list as an input. But your change shifts the indices of this list, so it would break

Reply:
[Note to self for future: It might be a good idea to change the targets into dicts so we can use named keys and stop referring to target items by their index in the code for the pipeline steps. Might speed up development]